jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/FilesTask.java

```diff
@@ -0,0 +1,174 @@
+/*
+ * Copyright 2018 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.google.cloud.tools.jib.gradle;
+
+import com.google.common.base.Preconditions;
+import java.io.File;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.HashSet;
+import java.util.Set;
+import javax.annotation.Nullable;
+import org.gradle.api.DefaultTask;
+import org.gradle.api.Project;
+import org.gradle.api.artifacts.Configuration;
+import org.gradle.api.artifacts.Dependency;
+import org.gradle.api.artifacts.ProjectDependency;
+import org.gradle.api.artifacts.PublishArtifact;
+import org.gradle.api.initialization.Settings;
+import org.gradle.api.plugins.JavaPluginConvention;
+import org.gradle.api.tasks.SourceSet;
+import org.gradle.api.tasks.TaskAction;
+
+/**
+ * Print out changing source dependencies on a project.
+ *
+ * <p>Expected use: {@code ./gradlew _jibSkaffoldFiles -q} or {@code ./gradlew
+ * :<subproject>:_jibSkaffoldFiles -q}
+ */
+public class FilesTask extends DefaultTask {
+
+  /**
+   * Prints the locations of a project's build.gradle, settings.gradle, and gradle.properties.
+   *
+   * @param project the project
+   */
+  private static void printGradleFiles(Project project) {
+    Path projectPath = project.getProjectDir().toPath();
+
+    // Print build.gradle
+    System.out.println(project.getBuildFile());
+
+    // Print settings.gradle
+    Path settingsFiles = projectPath.resolve(Settings.DEFAULT_SETTINGS_FILE);
```

Comment:
Naming: settingsFile**s** -> settingsFile
Also, don't expect a lot of customization here, but we can check if the settings file was configured on the command line.

```
getProject().getGradle().getStartParameter().getSettingsFile() != null
```
and use that instead of default.

Reply:
Is there an equivalent for gradle.properties?